src/cmd/src/cmdline.cpp

```diff
@@ -1448,7 +1423,48 @@ cmdline_parser_internal (
               goto failure;
           
           }
-          
+          else if (strcmp (long_options[option_index].name, "clear-results") == 0)
+          {
+            const int dir_err = system("rm -rf results/ 2> /dev/null");
```

Comment:
This will not work on Windows. Use this instead https://en.cppreference.com/w/cpp/filesystem/remove.

Reply:
Also, extract the results dir to a global variable if it's constant and reference it from all other functions that operate on it.

Reply:
Fede, no pude hacer andar la librería de <filesystem> por un problema de linkeo de archivos que no pude solucionar. Termine agregando el comando para windows para que ande de todas formas.